lib/booktype/apps/core/context_processors.py

```diff
@@ -0,0 +1,18 @@
+from django.conf import settings as django_settings
+from django.core.exceptions import ImproperlyConfigured
+
+
+# Inspired by https://github.com/mfogel/django-settings-context-processor
+def settings(request):
+    """
+    Adds the settings specified in settings.TEMPLATE_VISIBLE_SETTINGS to
+    the request context.
+    """
+    new_settings = {}
+    for attr in getattr(django_settings, "TEMPLATE_VISIBLE_SETTINGS", ()):
+        try:
+            new_settings[attr] = getattr(django_settings, attr)
```

Comment:
It would be cool to pull default values from `constants.py`, like using `config.get_configuration`. Although I'm ok with the way you have it now.